### PR TITLE
ISDK-1893 Settings to enable VP8 Simulcast while connecting to a Room

### PR DESCRIPTION
**Pull request by piyushtank**
Added settings to chose VP8 Simulcast while connecting to a Room - 

----
<kbd></kbd>
----
<kbd></kbd>

### CODE REVIEW

File: Utils/SettingsTableViewController.swift [hunk 1]
```diff
@@ -185,13 +185,25 @@ class SettingsTableViewController: UITableViewController {
...
+                                                style: .default,
+                                                handler: { (action) -> Void in
                 self.settings.videoCodec = codec
                 self.tableView.reloadData()
             })
             
             if (settings.videoCodec == codec) {
-                selectedButton = selectionButton;
+
+                // workaround for https://github.com/twilio/twilio-video-ios/issues/12
```
Comment (ceaglest) [inline Utils/SettingsTableViewController.swift]: Thanks for documenting the bug! Lets change - workaround -> Workaround

File: Utils/SettingsTableViewController.swift [hunk 2]
```diff
@@ -15,7 +15,7 @@ class SettingsTableViewController: UITableViewController {
     static let maxAudioBitrate = "Max Audio Bitrate (bps)"
     static let maxVideoBitrate = "Max Video Bitrate (bps)"
     static let defaultStr = "Default"
-    static let codecDisclaimerText = "Set your preferred audio and video codec. Not all codecs are supported with Group rooms. The media server will fallback to OPUS or VP8 if a preferred codec is not supported."
+    static let codecDisclaimerText = "Set your preferred audio and video codec. Not all codecs are supported with Group rooms. The media server will fallback to OPUS or VP8 if a preferred codec is not supported. VP8 Simulcast should only be enabled in a Group Room."
```
Comment (ceaglest) [inline Utils/SettingsTableViewController.swift]: We've got quite a few disclaimers here, I wonder if we could make this clearer for customers?

One sentence that I would change.

`Not all codecs are supported with Group rooms` -> `Not all codecs are supported in Group Rooms`.

File: Utils/SettingsTableViewController.swift [hunk 3]
```diff
@@ -15,7 +15,7 @@ class SettingsTableViewController: UITableViewController {
     static let maxAudioBitrate = "Max Audio Bitrate (bps)"
     static let maxVideoBitrate = "Max Video Bitrate (bps)"
     static let defaultStr = "Default"
-    static let codecDisclaimerText = "Set your preferred audio and video codec. Not all codecs are supported with Group rooms. The media server will fallback to OPUS or VP8 if a preferred codec is not supported."
+    static let codecDisclaimerText = "Set your preferred audio and video codec. Not all codecs are supported in Group rooms. The media server will fallback to OPUS or VP8 if a preferred codec is not supported. VP8 Simulcast should only be enabled in a Group Room."
```
Comment (ceaglest) [inline Utils/SettingsTableViewController.swift]: room -> Room

File: Utils/SettingsTableViewController.swift (hunk 2 above)
Comment (piyushtank) [reply Utils/SettingsTableViewController.swift]: I agree with having a cluster of disclaimers. I am open for suggestions..

**Pull request merged**